marketplace-client/src/marketplace/src/app/account/register-stuff/register-stuff.component.ts

```diff
@@ -0,0 +1,94 @@
+import {Component, OnInit} from '@angular/core';
+import {AbstractControl, FormBuilder, FormGroup, Validators} from "@angular/forms";
+import {AccountService} from "../../_services/account.service";
+import {validateBirthday} from "../../_helpers/validators.service";
+import {first} from "rxjs/operators";
+import {Role} from "../../_models/role";
+
+@Component({
+  selector: 'app-courier',
```

Comment:
it's not ok when a selector name is different from a component name

Reply:
I haven't noticed this when I was renaming the whole component.